### ISSUE TITLE
Digiline injectors improvements

### ISSUE THREAD

**Issue by Klaranth** [pandorabox-io/in-game#29]
Houlley — 04/19/2021
digiline injectors can have option to specify what inventory it can pull items from, if the container has multiple inventory types; eg furnace has fuel, input and output

1hittoaster — 04/19/2021
I'd love to be able to purge an auto-crafter's input slots....

<details><summary><i>Unrelated stuff</i></summary>
SX — 04/21/2021
Could it be possible to increase active_object_send_range_blocks to 3 without affecting performance too much? When active_block_range is smaller than active_object_send_range_blocks it will actually use players current view direction to actually collect objects farther than active_block_range.
Basically if active_block_range = 1 and active_object_send_range_blocks = 3 then active objects 3 blocks away in front of player will be sent but behind player only 1 block worth of objects will be collected.
While not much this would allow seeing trains :advtrains_japan: / mobs :rabbit2: / items in tubes :face_vomiting: bit further away.

BuckarooBanzai — 04/21/2021
last time i played with that it somehow also increased the active-mapblock setting somehow, but yeah: go ahead, we'll find out if it works properly this time :fingers_crossed:
</details>

**Comment by Klaranth** [comment]
Nothing new added to this issue for 12 days. As being non-trivial, this will be closed for now.
